lib/pubsub/topic.js

```diff
@@ -144,11 +147,17 @@ Topic.prototype.publish = function(messages, callback) {
   callback = callback || util.noop;
 
   var body = {
-    topic: this.name,
     messages: messages.map(Topic.formatMessage_)
   };
 
-  this.makeReq_('POST', 'topics/publishBatch', null, body, callback);
+  var path = this.name + ':publish';
```

Comment:
@tmatsuo Any idea why this `:publish` style for API URLs is used? Personally, I kinda like it, but perhaps you can fill us in on the reasoning for using it?

It feels very much like I'm making a method call. The resource is `/projects/my-project/topics/my-topic` and my action on that resource is `:publish` :smiley: